plugins/CoreAdminHome/templates/generalSettings.twig

```diff
@@ -1,290 +1,226 @@
 {% extends 'admin.twig' %}
 
 {% block content %}
-{# load macros #}
-{% import 'macros.twig' as piwik %}
-{% import 'ajaxMacros.twig' as ajax %}
+    {% import 'macros.twig' as piwik %}
+    {% import 'ajaxMacros.twig' as ajax %}
 
-{% if isSuperUser %}
```

Comment:
Yes I removed this as it was duplicating the `check...` in the controller (this template is used only there). Thanks for reviewing that though because this extra check made me doubt to remove it ;)